src/components/MainTabs/index.jsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+import AppBar from '@material-ui/core/AppBar';
+import Tabs from '@material-ui/core/Tabs';
+import Tab from '@material-ui/core/Tab';
+import Typography from '@material-ui/core/Typography';
+import BugzillaComponents from '../BugzillaComponents';
+import Reportees from '../Reportees';
+
+function TabContainer(props) {
+  const { children } = props;
+  return (
+    <Typography component="div" style={{ padding: 8 * 3 }}>
+      {children}
+    </Typography>
+  );
+}
+
+TabContainer.propTypes = {
+  children: PropTypes.node.isRequired,
+};
+
+const styles = theme => ({
+  root: {
+    flexGrow: 1,
+    backgroundColor: theme.palette.background.paper,
+  },
+});
+
+class MainTabs extends React.Component {
+    state = {
+      value: 0,
+    };
+
+    handleChange = (event, value) => {
+      this.setState({ value });
+    };
+
+    render() {
+      const {
+        classes, ldapEmail, partialOrg, onPersonDetails, teamComponents,
+        bugzillaComponents, onComponentDetails,
+      } = this.props;
+      const { value } = this.state;
+
+      return (
+        <div className={classes.root}>
+          <AppBar position="static">
+            <Tabs value={value} onChange={this.handleChange}>
+              <Tab label="People" />
+              <Tab label="Teams" />
+              <Tab label="Components" />
+            </Tabs>
+          </AppBar>
+          {value === 0 && (
+            <TabContainer>
+              <Reportees
+                ldapEmail={ldapEmail}
+                partialOrg={partialOrg}
+                onPersonDetails={onPersonDetails}
+              />
+            </TabContainer>
+          )}
+
+          {value === 1 && (
+            <TabContainer>
+              <BugzillaComponents
+                title="Teams"
```

Comment:
With the tabs there's no need to specify a title.

Also change the code that adds 'Reportees':
<img width="126" alt="image" src="https://user-images.githubusercontent.com/44410/54712550-f00f0c00-4b22-11e9-8cfa-22d917c6f524.png">

Reply:
I removed the code related to Title in BugzillaComponents and Reportees also, hope that's ok.